tests/pkg_tweetnacl/main.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       tweetnacl NaCl crypto library example
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include <tweetnacl.h>
+#include "ps.h"
+
+static const char message[] = "0123456789abcdef";
+
+#define MLEN (sizeof(message) + crypto_box_ZEROBYTES)
+
+static unsigned char alice_pk[crypto_box_PUBLICKEYBYTES];
+static unsigned char alice_sk[crypto_box_SECRETKEYBYTES];
+static unsigned char bob_pk[crypto_box_PUBLICKEYBYTES];
+static unsigned char bob_sk[crypto_box_SECRETKEYBYTES];
+static unsigned char m[MLEN];
+static unsigned char c[MLEN];
+static const unsigned char n[crypto_box_NONCEBYTES];
+
+static unsigned char result[MLEN];
+
+int main(void)
+{
+    int res;
+
+    printf("Creating keypair ALICE...\n");
+    crypto_box_keypair(alice_pk, alice_sk);
```

Comment:
And what's the purpose of such test? I can imagine that it's supposed to be run by the CI on each PR to check if it's not breaking any functionality on the use of this library, but as I stated before, the library does not depend at all on any RIOT code, so no PR on the RIOT side can break it.